src/pharmpy/model/statements.py

```diff
@@ -1113,7 +1130,10 @@ def _find_depot(self):
         depot = None
         for to_central, _ in self.get_compartment_inflows(central):
             outflows = self.get_compartment_outflows(to_central)
-            if len(outflows) == 1:
+            if len(outflows) == 1 or len(outflows) == 2:
+                if len(outflows) == 2:
+                    if not self.get_flow(to_central, self.find_metabolite()):
```

Comment:
Why does the logi for finding the depot need to be changed?

Reply:
This is due to presystemic metabolite models where the depot then will have two outflows (as opposed to one). One going to the central compartment and one going to the metabolite compartment.
As it was previously defined as a compartment having a single outflow to the central compartment, this would be incorrect for presystemic metabolite models